libs/styles/styles.css

```diff
@@ -74,13 +74,19 @@
   --spacing-xs: 16px;
   --spacing-s: 24px;
   --spacing-m: 32px;
-  --spacing-l: 48px;
-  --spacing-xl: 56px;
-  --spacing-xxl: 80px;
+  --spacing-l: 40px;
+  --spacing-xl: 48px;
+  --spacing-xxl: 56px;
+  --spacing-xxxl: 80px;
 
   /* grid sizes */
   --grid-container-width: 83.4%;
   --grid-column-width: calc(var(--grid-container-width) / 12);
+
+  /* block container */
+  --block-container-width-s: 800px;
```

Comment:
Where is this coming from? I know John Spencer has said stuff like, "This is 8 columns" which would be `calc(var(--grid-container-width) / 8);`. Not sure how feasible this is, but worth asking about. 

Also seems a little dangerous to have this in the "mobile / default" styles because 800px is way bigger than a mobile viewport.

Reply:
These are the max widths for a media s/m/l containers. 
I did removed these global vars as they aren't widely used and refactored their respective blocks. I ended up keeping them as set px values in .media so they co-operate w/ the wrapped .container.